ts/Dashboards/Components/Component.ts

```diff
@@ -1144,6 +1154,7 @@ namespace Component {
             detail?: AnyRecord;
         } & EventRecord;
 
+    /** @internal */
     export type SyncOptions = Record<string, boolean | Partial<Sync.OptionsEntry>>;
```

Comment:
Shouldn't be `SyncOptions` available?

![image](https://user-images.githubusercontent.com/1453926/227250122-c6cb915c-58a2-4448-962c-03cf97fa187d.png)

Reply:
Might be connected: https://github.com/highcharts/highcharts/pull/18723
@goransle